CHANGELOG.md

```diff
@@ -13,12 +24,12 @@
 
 ### Changes
 
-* [BUGFIX]
-* [BUGFIX] Fix node_textfile_mtime_seconds to work properly on symlinks #1326
-* [CHANGE] Renamed `interface` label to `device` in netclass collector #1224
 * [BUGFIX] Add fallback for missing /proc/1/mounts #1172
+* [BUGFIX] Fix node_textfile_mtime_seconds to work properly on symlinks #1326
```

Comment:
CHANGE, FEATURE, ENHANCEMENT, BUGFIX is the order

Reply:
Done